server/resolvers/author/mutation.js

```diff
@@ -0,0 +1,16 @@
+const { Author, Book } = require("../../models");
+
+const AuthorMutation = {
+  addAuthor: ({ name, biography, born_date }) =>
+    Author.create({ name, biography, born_date }),
+  updateAuthor: async ({ id, ...updates }) => {
```

Comment:
say `Author` has another field called AuthorNo. that is not something we allow to update from frontend, then anyway to ensure that the updates are cleaned up before passing it to the model ?

Reply:
You're correct, should've done data validations